### ISSUE TITLE
F3, Find Next, acts like "Find Under Cursor"

### ISSUE THREAD

**Issue by kasajian** [Microsoft/vscode#1857]
tl; dr: F3 acts like Ctrl+F3 in Visual Studio

##### More details:

Let's say the text editor contains the following 4 lines:

    ABC
    ABC
    XYZ
    ABC

The cursor is at the very top, of the file, at the first ABC.  I hit Ctrl+F to show the Find dialog.  I type ABC.  The first ABC is highlighted.  I hit Esc to exit the Find dialog.
The cursor is now at end of the first line, with ABC on that line highlighted.  I hit delete to remove it and change the text to XYZ.   At this point the text editor looks like this:

    XYZ
    ABC
    XYZ
    ABC

The cursor is at end of the first line.

I hit F3 to "Find Next" to find the next occurrence of ABC, but instead it searches for XYZ.  The reason is because hitting F3 changes what's being searched for.   In other words, F3 acts like Ctrl+F3 in Visual Studio

##### Version I'm running:
[Window Title]
Visual Studio Code

[Main Instruction]
Visual Studio Code

[Content]

Version 0.10.6
Commit a80232bbcfe8a5cdad1ebc98638673d9dcb02458
Date 2015-12-19T11:34:58.895Z
Shell 0.34.5
Renderer 45.0.2454.85
Node 4.1.1

[OK]

**Comment by Tyriar** [comment]
Nice catch! Looks like <kbd>F3</kbd> is probably doing whatever <kbd>ctrl</kbd>+<kbd>f</kbd> is doing to pull in the word the cursor is on before executing a find.

**Comment by alexandrudima** [comment]
@kasajian Thank you for the detailed steps
